### PR TITLE
update kafka onboarding docs

### PR DESCRIPTION
**Pull request by 4n4nd**
Resolves https://github.com/operate-first/apps/issues/804

### CODE REVIEW

File: docs/odh/kafka/add_kafka_users.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Then add it to `odh-manifests/kafka/overlays/users/kustomization.yaml` by running the following:
+
+```bash
+$ cd odh-manifests/kafka/overlays/users
+$ kustomize edit add resource my-user.yaml
+```
+
+If you don't have `kustomize` then simply add this file manually.
+
+The steps are identical for the other environments.
```
Comment (HumairAK) [inline docs/odh/kafka/add_kafka_users.md:66]: There are no other environments `odh-manifests` folder is purely for the zero cluster deployment of odh. Let's remove this part.

Comment (HumairAK) [reply docs/odh/kafka/add_kafka_users.md:66]: same for the other file

Comment (tumido) [reply docs/odh/kafka/add_kafka_users.md:66]: @4n4nd, @HumairAK was this addressed and resolved?

Comment (HumairAK) [reply docs/odh/kafka/add_kafka_users.md:66]: I still see it there, so no.

Comment (HumairAK) [reply docs/odh/kafka/add_kafka_users.md:66]: @4n4nd ^

**Comment by HumairAK** [comment]
bump

File: docs/odh/kafka/add_kafka_users.md (hunk 1 above)
Comment (4n4nd) [reply docs/odh/kafka/add_kafka_users.md]: updated :+1:

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Pull request merged**